.github/workflows/build_dependencies.yml

```diff
@@ -233,6 +232,7 @@ jobs:
     - name: Code Coverage Run
       run: |
         pre=$([[ "${{ inputs.build-type }}" != "Debug" ]] && echo "-o sisl:prerelease=${{ inputs.prerelease }}" || echo "")
+        sudo rm -rf $ANDROID_HOME
```

Comment:
in the failure CI , `Create and Test Package` is not scheduled ,so `sudo rm -rf /usr/local/lib/android/` will not be executed.

here , we add this to `Code Coverage Run` will be definitely scheduled in every CI pipeline

@sanebay